app/models/user.rb

```diff
@@ -17,6 +17,10 @@ class User < ApplicationRecord
   validates :last_name, presence: true
   validates :role, inclusion: { in: ROLES }
 
+  def name
+    [ first_name, last_name ].compact.join(" ")
+  end
```

Comment:
Rather than do a type conversion to an array and then back to a string, we can just interpolate:

```suggestion
  def name
    "#{first_name} #{last_name}"
  end
```